compose.yaml

```diff
@@ -40,6 +40,8 @@ services:
         "
     volumes:
       - ./alfresco/resources/openannotate-override-placeholders.properties:/usr/local/tomcat/shared/classes/openannotate-override-placeholders.properties
+      - ./alfresco/resources/opencontent-override-placeholders.properties:/usr/local/tomcat/shared/classes/alfresco/module/com.tsgrp.opencontent/opencontent-override-placeholders.properties
+      - ./alfresco/resources/customTranformer.json:/usr/local/tomcat/shared/classes/alfresco/extension/transform/pipelines/customTranformer.json
```

Comment:
This should be `customTransformer.json` instead of `customTranforme.json`, right?

Reply:
Yes this should be customTranformer.json

Reply:
Please, fix the typo: it should be `customTransformer.json`, with the "s" before "f" and after "n"

Reply:
Corrected in updated Commit.